graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -77,9 +78,16 @@ def _get_fields_for_class(schema_graph, graphql_types, field_type_overrides, hid
         property_name: graphql_type
         for property_name, graphql_type in six.iteritems(all_properties)
         if not (isinstance(graphql_type, GraphQLList) and
-                isinstance(graphql_type.of_type, GraphQLObjectType))
+                not isinstance(graphql_type.of_type, GraphQLScalarType))
     }
 
+    class_collection_properties = set(all_properties.keys()).difference(result.keys())
+    if len(class_collection_properties) > 0:
+        warnings.warn('The fields {} of class {} were ignored since they are GraphQLLists of '
```

Comment:
In the test, can we assert that this warning gets raised?

Reply:
Seems reasonable. I will add this in a test in the next pr update.

Reply:
python2's unittest.Test is missing an self.assertWarns method. So adding this test is a bit problematic. The effort to figure out how to add this test does not seem to match the reward so I've decided not to do it.